packages/vscode-extension/src/debugging/templates/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./getArraySlots";
```

Comment:
do we really need this file?

Reply:
Nah, I added this assuming there might be more in the future and it's better to import from one place. Remove in https://github.com/software-mansion/radon-ide/pull/846/commits/57750e9475178114cf6c386c4a97df2cb75d8682